content/en/docs/ambient/overview/index.md

```diff
@@ -26,7 +26,7 @@ For details on the design of ambient mode, and how it interacts with the Istio {
 
 The ztunnel (Zero Trust tunnel) component is a purpose-built, per-node proxy that powers Istio's ambient data plane mode.
 
-Ztunnel is responsible for securely connecting and authenticating workloads within the mesh. The ztunnel proxy is written in Rust and is intentionally scoped to handle L3 and L4 functions such as mTLS, authentication, L4 authorization and telemetry. Ztunnel does not terminate workload HTTP traffic or parse workload HTTP headers. The ztunnel ensures L3 and L4 traffic is efficiently and securely transported either directly to other pods' ztunnel proxies or to waypoint proxies, where the full suite of Istio’s L7 functionality, such as HTTP telemetry and load balancing, is implemented.
+Ztunnel is responsible for securely connecting and authenticating workloads within the mesh. The ztunnel proxy is written in Rust and is intentionally scoped to handle L3 and L4 functions such as mTLS, authentication, L4 authorization and telemetry. Ztunnel does not terminate workload HTTP traffic or parse workload HTTP headers. The ztunnel ensures L3 and L4 traffic is efficiently and securely transported either directly to other pods' ztunnel proxies or to waypoint proxies. Waypoint proxies are used when the full suite of Istio’s L7 functionality, such as HTTP telemetry and load balancing, are needed.
```

Comment:
```suggestion
Ztunnel is responsible for securely connecting and authenticating workloads within the mesh. The ztunnel proxy is written in Rust and is intentionally scoped to handle L3 and L4 functions such as mTLS, authentication, L4 authorization and telemetry. Ztunnel does not terminate workload HTTP traffic or parse workload HTTP headers. The ztunnel ensures L3 and L4 traffic is efficiently and securely transported either directly to other pods' ztunnel proxies or to waypoint proxies. Waypoint proxies can be used when the full suite of Istio’s L7 functionality, such as HTTP telemetry and load balancing, are needed.
```
Saying "are used" makes it sound a little bit like something istiod or ztunnel detects and then automatically uses for you. "can be used", "can be added" or even "should be added/used" would perhaps be clearer.